Mathlib/RingTheory/MvPowerSeries/Basic.lean

```diff
@@ -19,6 +21,34 @@ A formal power series is to a polynomial like an infinite sum is to a finite sum
 
 We provide the natural inclusion from multivariate polynomials to multivariate formal power series.
 
+## Main definitions
```

Comment:
I'm happy with the changes to this file. If you split it into a separate PR, then it could be merged already.

Reply:
In fact, it was already merged a few days ago, I wonder how the `merge master` did spot a difference here.